packages/core/entity/FileExplorerURL/index.ts

```diff
@@ -72,8 +74,11 @@ export const DEFAULT_AICS_FMS_QUERY: FileExplorerURLComponents = {
 export const getNameAndTypeFromSourceUrl = (dataSourceURL: string) => {
     const uriResource = dataSourceURL.substring(dataSourceURL.lastIndexOf("/") + 1).split("?")[0];
     const name = `${uriResource} (${new Date().toLocaleDateString()} ${new Date().toLocaleTimeString()})`;
```

Comment:
I don't understand the repeated date thing, but if its intentional, then it is probably OK?

Reply:
One is for date and one is for time; I suspect that the reason they're generated separately is that `toLocaleString()` returns them with `, ` in between and we just want a space apparently?